### PR TITLE
chore: refactor --help texts

### PR DESCRIPTION
**Pull request by mariux**

### CODE REVIEW

File: cmd/terramate/cli/cli.go [hunk 1]
```diff
@@ -109,144 +109,140 @@ type kongParallelFlag struct {
...
+	VersionFlag    bool     `hidden:"true" name:"version" help:"Show Terramate version."`
+	Chdir          string   `short:"C" optional:"true" predictor:"file" help:"Set working directory."`
+	GitChangeBase  string   `short:"B" optional:"true" help:"Set git base reference for computing changes."`
+	Changed        bool     `short:"c" optional:"true" help:"Filter stacks based on changes made in git."`
+	Tags           []string `optional:"true" sep:"none" help:"Filter stacks by tags."`
+	NoTags         []string `optional:"true" sep:"," help:"Filter stacks by tags not being set."`
+	LogLevel       string   `optional:"true" default:"warn" enum:"disabled,trace,debug,info,warn,error,fatal" help:"Log level to use: 'disabled', 'trace', 'debug', 'info', 'warn', 'error', or 'fatal'."`
+	LogFmt         string   `optional:"true" default:"console" enum:"console,text,json" help:"Log format to use: 'console', 'text', or 'json'."`
+	LogDestination string   `optional:"true" default:"stderr" enum:"stderr,stdout" help:"Destination channel of log messages: 'stderr' or 'stdout'."`
+	Quiet          bool     `optional:"false" help:"Disable outputs."`
```
Comment (i4ki) [inline cmd/terramate/cli/cli.go:121]: The `--quiet` is at the moment behaving as "disable helper messages", as it only disables the messages like "Entering directory...", "Executing ...", etc, but the actual output of the command is still printing.

This flag is heavily used in the e2e tests, so we hide all non-important messages and just assert for the important output but the `--help` for it is a bit misleading, as it is disabling all outputs.

File: cmd/terramate/cli/cli.go [hunk 2]
```diff
@@ -109,144 +109,140 @@ type kongParallelFlag struct {
...
-		EnsureStackIds bool     `help:"generate an UUID for the stack.id of all stacks which does not define it"`
-		NoGenerate     bool     `help:"Disable code generation for the newly created stacks"`
-	} `cmd:"" help:"Creates a stack on the project"`
+		Path           string   `arg:"" optional:"" name:"path" predictor:"file" help:"Path of the new stack."`
+		ID             string   `help:"Set the ID of the stack, defaults to an UUIDv4 string."`
+		Name           string   `help:"Set the name of the stack, defaults to the basename of <path>"`
+		Description    string   `help:"Set the description of the stack, defaults to <name>"`
+		Import         []string `help:"Add 'import' block to the configuration of the new stack."`
+		After          []string `help:"Add 'after' attritbute to the configuration of the new stack."`
+		Before         []string `help:"Add 'before' attritbute to the configuration of the new stack."`
```
Comment (i4ki) [inline cmd/terramate/cli/cli.go:136]:
```suggestion
		Before         []string `help:"Add 'before' attribute to the configuration of the new stack."`
```

File: cmd/terramate/cli/cli.go [hunk 3]
```diff
@@ -109,144 +109,140 @@ type kongParallelFlag struct {
...
-		AllTerragrunt  bool     `help:"initialize all Terragrunt modules"`
-		EnsureStackIds bool     `help:"generate an UUID for the stack.id of all stacks which does not define it"`
-		NoGenerate     bool     `help:"Disable code generation for the newly created stacks"`
-	} `cmd:"" help:"Creates a stack on the project"`
+		Path           string   `arg:"" optional:"" name:"path" predictor:"file" help:"Path of the new stack."`
+		ID             string   `help:"Set the ID of the stack, defaults to an UUIDv4 string."`
+		Name           string   `help:"Set the name of the stack, defaults to the basename of <path>"`
+		Description    string   `help:"Set the description of the stack, defaults to <name>"`
+		Import         []string `help:"Add 'import' block to the configuration of the new stack."`
+		After          []string `help:"Add 'after' attritbute to the configuration of the new stack."`
```
Comment (i4ki) [inline cmd/terramate/cli/cli.go:135]:
```suggestion
		After          []string `help:"Add 'after' attribute to the configuration of the new stack."`
```

**Review by i4ki — CHANGES_REQUESTED** [review]

File: cmd/terramate/cli/cli.go (hunk 1 above)
Comment (mariux) [reply cmd/terramate/cli/cli.go:121]: this is fine. we did not say we disable outputs of the commands executed.

**Review by i4ki — APPROVED** [review]
LGTM

**Pull request merged**